### DISCUSSION TITLE
Can't wake up Miles

### DISCUSSION THREAD

**Discussion by Kraalin** [small-cactus/M.I.L.E.S#31] [General]
Hi, i can't manage to wake up Miles. From the powershell i can see the ''this is a test'' config with right answer complete. But after that, even after full restart, i'm not able to wake up Miles. On the power shell the i got an Deprecation warning error about CGI. 
Now i can see : Python Output: Listening for 'Miles' ...
and nothing more. I was thinking since i'm french i may pronouce bad, but with 50k training i doubt. Any clues about what i mess ? :)

**Comment by small-cactus** [comment]
All of the training data is various English accents and very rare occurrences of other accents. It's very unlikely that it just doesn't work if the speech recognition was successful at startup.

Your best bet would be to channel your inner American and really pronounce it as best as you possibly can

**Reply by small-cactus** [reply]
Looking more into it, I'm 80% sure it's the French pronunciation.

"Miles" is pronounced as "myles," rhyming with "smiles."

For French, I believe this might help you say it correctly, try saying:

“Maïlz”

**Reply by Kraalin** [reply]
hi. i'm still unable. is there any way i can bypass the wake up command to test if he can process what i'm saying. Other then the This i s a test in my powershell the other day i see nothing moving :O 
 I also see that the .script is not present  in the Miles-V2 Folder here a screen shot of my powershell
![image](https://github.com/user-attachments/assets/e8b5fb8d-eaad-4803-948f-3db03c45acee)

**Reply by small-cactus** [reply]
Go to control panel and tell me what sample rate your microphone is set to, Miles only works with a sample rate of 41000hz input. Miles also does not work at all with heavy background noise like air conditioning or loud fans.
